playground/app/procedureCodes/index.js

```diff
@@ -6,73 +6,12 @@ export default Object.assign({}, schema, uiSchema, rules, {
   formData: {
     procedureCodes: [
       {
-        diagnosis: {
```

Comment:
please keep this json to load precedurecode plugin with features.

Reply:
Added back